src/_includes/soft-mod-cheat-sheet-banner.html

```diff
@@ -2,7 +2,9 @@
 {% assign background_position = 'top left' %}
 {% capture title %}{% t software-modernisation.cheat-sheet-cta-banner-title %}{% endcapture %}
 {% capture description %}{% t software-modernisation.cheat-sheet-cta-banner-text %}{% endcapture %}
-{% capture cta_href %}{% t software-modernisation.cheat-sheet-cta-href %}{% endcapture %}
 {% capture cta_text %}{% t software-modernisation.cheat-sheet-cta-banner-link %}{% endcapture %}
-{% capture hubspot-cta %}{%- include soft_mod_hubspot_cheat_sheet_banner_cta.liquid -%}{% endcapture %}
+{%- capture hubspot-id -%} 
+  {%- if site.lang == 'es' -%}041e3783-91f3-4bc1-8926-7376fa4faae0{%- else -%}06e1ce72-a796-4e1c-8ff1-1b9c7a67fe57{%- endif -%}
+{%- endcapture -%}
+{% capture hubspot-cta %}{%- include hubspot_cta.liquid class="hubspot-in-page-banner" hubspot-id=hubspot-id -%}{% endcapture %}
 {% include in_page_banner.html background_image=background_image background_position=background_position class="soft-mod-cheat-sheet-cta-banner" cta_href=cta_href cta_text=cta_text description=description title=title hubspot-cta=hubspot-cta split="right-heavy" %}
```

Comment:
we can remove the `cta_href` and `cta_text` properties on this include I think, as they are handled by the hubspot cta

Reply:
Done :)